e2e/tests/init_test.go

```diff
@@ -76,20 +95,108 @@ func (s *suite) TheUserSendsUpoktToAnotherAddress(amount int64) {
 	s.pocketd.result = res
 }
 
-func (s *suite) getAddresses() [2]string {
-	var strs [2]string
+func (s *suite) TheAccountHasABalanceGreaterThanUpokt(acc string, amount int64) {
+	s.buildAddrMap()
+	args := []string{
+		"query",
+		"bank",
+		"balances",
+		accNameToAddr[acc],
+	}
+	res, err := s.pocketd.RunCommandOnHost("", args...)
+	if err != nil {
+		s.Fatalf("error getting balance: %s", err)
+	}
+	s.pocketd.result = res
+	match := amountRe.FindStringSubmatch(res.Stdout)
+	if len(match) < 2 {
+		s.Fatalf("no balance found for %s", acc)
+	}
+	found, err := strconv.Atoi(match[1])
+	require.NoError(s, err)
+	if int64(found) < amount {
+		s.Fatalf("account %s does not have enough upokt: %d < %d", acc, found, amount)
+	}
+	s.tempState[acc] = found // save the balance for later
+}
+
+func (s *suite) TheAccountBalanceIsKnownFor(acc string) {
+	s.buildAddrMap()
+	args := []string{
+		"query",
+		"bank",
+		"balances",
+		accNameToAddr[acc],
+	}
+	res, err := s.pocketd.RunCommandOnHost("", args...)
+	if err != nil {
+		s.Fatalf("error getting balance: %s", err)
+	}
+	s.pocketd.result = res
+	match := amountRe.FindStringSubmatch(res.Stdout)
+	if len(match) < 2 {
+		s.Fatalf("no balance found for %s", acc)
+	}
+	found, err := strconv.Atoi(match[1])
+	require.NoError(s, err)
+	s.tempState[acc] = found // save the balance for later
+}
+
+func (s *suite) TheAccountBalanceOfShouldBeUpoktThanBefore(acc string, amount int64, condition string) {
+	s.buildAddrMap()
+	prev, ok := s.tempState[acc]
+	if !ok {
+		s.Fatalf("no previous balance found for %s", acc)
+	}
+	args := []string{
+		"query",
+		"bank",
+		"balances",
+		accNameToAddr[acc],
+	}
+	res, err := s.pocketd.RunCommandOnHost("", args...)
+	if err != nil {
+		s.Fatalf("error getting balance: %s", err)
+	}
+	s.pocketd.result = res
+	match := amountRe.FindStringSubmatch(res.Stdout)
+	if len(match) < 2 {
+		s.Fatalf("no balance found for %s", acc)
+	}
+	found, err := strconv.Atoi(match[1])
```

Comment:
The entire code block for doing a `query bank balances accAddr` and getting back and `int` can be factored out into a helper funciton.